.eslintrc.js

```diff
@@ -4,17 +4,20 @@ module.exports = {
     'es2021': true,
   },
   'extends': [
-    'google',
+    'plugin:@typescript-eslint/recommended',
   ],
   'parser': '@typescript-eslint/parser',
   'parserOptions': {
     'ecmaVersion': 'latest',
     'sourceType': 'module',
   },
   'plugins': [
-    '@typescript-eslint',
+    '@typescript-eslint/eslint-plugin',
+    'eslint-plugin-tsdoc',
   ],
   'rules': {
     'max-len': 'off',
+    'tsdoc/syntax': 'warn',
+    '@typescript-eslint/no-var-requires': 0,
```

Comment:
I also add
`'@typescript-eslint/explicit-function-return-type': 'warn',`
but it's a matter of taste

Reply:
Done! I added this and return types for all the functions.